### ISSUE TITLE
TypeScript configuration docs link to incorrect version of Fork-TS-Checker-Webpack-Plugin

### ISSUE THREAD

**Issue by jldillman** [storybookjs/storybook#11804]
The "See docs" link in Storybook v6's TypeScript configuration guide points to the wrong version of `fork-ts-checker-webpack-plugin`:
https://github.com/storybookjs/storybook/blob/next/docs/src/pages/configurations/typescript-config/index.md#mainjs-configuration

Storybook v6's `package.json` declares `"fork-ts-checker-webpack-plugin": "^4.1.4"` [here](https://github.com/storybookjs/storybook/blob/next/lib/core/package.json#L92), but the link points to the most recent version, which is `v5.0.14`.  Fork-TS-Checker's configuration changed from v4 -> v5, so the docs in question provide a configuration shape that is not compatible with the version used by Storybook.

Can the docs be updated to link to [this location](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/v4.1.6/README.md#options) instead?  This might help prevent confusion for those not overly-familiar with Fork-TS-Checker or the changes from v4 -> v5.

Screenshot:
![image](https://user-images.githubusercontent.com/19934927/89346349-81c2df00-d66e-11ea-87d0-7907e7a89342.png)

**Comment by jldillman** [comment]
Also, it might be useful to provide an example shape for custom configurations of `fork-ts-checker-webpack-plugin`, ie:

```
module.exports = {
  typescript: {
    check: true,
    checkOptions: {
      compilerOptions: {
        // skip checking *.d.ts
        skipLibCheck: true,
      },
    },
  },
};
```

**Comment by shilman** [comment]
@jldillman thanks for the heads up! will fix

@mrmckeb maybe we should upgrade `fork-ts-checker-webpack-plugin` before v6 goes out since it'll be a breaking change after? WDYT?

**Comment by krislao** [comment]
@jldillman thanks for reporting this issue! i got stuck following the storybook v6 docs to enable eslint but it's not working. now it's working using the v4.1.6 options. hope the docs will be updated soon! 🤞

**Comment by HJain13** [comment]
Hey @shilman, can I take this up?

**Comment by HJain13** [comment]
I forgot to mark the PR as one closing this, and as it was merged it can be manually closed now, I guess

**Comment by maneetgoyal** [comment]
@HJain13 The link on this [docs page](https://storybook.js.org/docs/react/configure/typescript) takes me to: 

https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/v4.1.6/README#options

instead of

https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/v4.1.6/README.md#options.

Should I be waiting for a new release? 

cc @shilman

**Comment by jonniebigodes** [comment]
@maneetgoyal sorry for the issue you're experiencing and for taking a bit longer than expected. I did a sweep of the broken links still present in the documentation and this is one of them. I've submitted a pull request [#12716](https://github.com/storybookjs/storybook/pull/12716), that fixes this issue. Once it's merged the link will display the proper README file for the package, while version 5.x is not implemented in Storybook. 

Stay safe
